### PR TITLE
Remove int.TryParse

### PR DESCRIPTION
**Pull request by dicko2**
As per @oznetmaster and @CharliePoole 's comments on the merge, this is to fix the CF build

### CODE REVIEW

File: src/NUnitFramework/framework/Internal/Execution/CompositeWorkItem.cs [hunk 1]
```diff
@@ -279,8 +279,8 @@ public int Compare(WorkItem x, WorkItem y)
             {
                 var xKey = int.MaxValue;
                 var yKey = int.MaxValue;
-                if (x.Test.Properties.ContainsKey(PropertyNames.Order)) int.TryParse(x.Test.Properties[PropertyNames.Order][0].ToString(), out xKey);
-                if (y.Test.Properties.ContainsKey(PropertyNames.Order)) int.TryParse(y.Test.Properties[PropertyNames.Order][0].ToString(), out yKey);
+                if (x.Test.Properties.ContainsKey(PropertyNames.Order)) xKey=(int)x.Test.Properties[PropertyNames.Order][0];
```
Comment (oznetmaster) [inline src/NUnitFramework/framework/Internal/Execution/CompositeWorkItem.cs]: I thought that the coding standards indicated that single line if statements were to be discouraged?  Perhaps I am mistaken?

**Comment by oznetmaster** [comment]
Aside from my comment above, this looks ok to merge.

**Comment by dicko2** [comment]
fixed up. Have you guys thought about using something like stylecop so you can easily enforce rules?

**Comment by oznetmaster** [comment]
There have been some interesting discussions  on "enforcing" coding "rules" :)  Perhaps @CharliePoole can point you to them.

**Comment by CharliePoole** [comment]
@dicko2 StyleCop would be handy if we took the time to create rules for our standards.

I  think @oznetmaster is referring to my dislike of utilities that actually reformat the code rather than issuing warnings. :-)

"Some day when I have time" I'd like to develop a form of standard that gives differing "weights" to different rules.

**Comment by CharliePoole** [comment]
@dicko2 Please contact me offline regarding converting you from a contributor to a committer. charliepoole AT gmail DOT com

**Pull request merged**